src/main/java/org/folio/linked/data/configuration/json/deserialization/event/SourceRecordDomainEventDeserializer.java

```diff
@@ -44,9 +46,24 @@ public SourceRecordDomainEvent deserialize(JsonParser jp, DeserializationContext
     if (node.has(EVENT_PAYLOAD)) {
       event.setEventPayload(getSourceRecord(node.get(EVENT_PAYLOAD)));
     }
+    if (node.has(EVENT_METADATA)) {
```

Comment:
We should not use the date coming in the event. Instead, we should use the current date and user.